party.py

```diff
@@ -0,0 +1,147 @@
+# ------------- WAVE 1 --------------------
+
+from tests.test_constants import GENRE_1, MOVIE_TITLE_1, RATING_1

```

Comment:
In general, we don't want to import values from a test file into our main python files. For our functions in party.py we won't need values from GENRE_1, MOVIE_TITLE_1, RATING_1. We can get our data from the parameters.